.local/bin/workman

```diff
@@ -225,11 +267,7 @@ assign() {
     # Attach if the task is already assigned.
     ASSIGNED="$(_find_assigned_for_task ${TASK})"
     if [ ! -z $ASSIGNED ]; then
-        echo "${BLUE}::${RESET} Found existing working directory for \"${TASK}\", attaching..."
-        pushd "${ASSIGNED}"
-        tmux new -A -s "${PROJECT_NAME}/${TASK}"
-        popd
-        exit 0
+        _open $TASK $ASSIGNED
     fi
```

Comment:
I've just realized you were totally right about the `_load_config()` function - if we define the defaults at the top then we can't easily re-source them so that they have values for `TASK` in functions like this (which previously wasn't an issue because every configuration option was in the config file). Sorry.

We'll need to unrevert that and add a call in this function so `SESSION_ATTACH_COMMAND` has a value with `TASK` set.

Reply:
i rebased and removed the revert commit